src/assets/wise5/components/table/tableController.ts

```diff
@@ -276,12 +297,68 @@ class TableController extends ComponentController {
       this.dataExplorerGraphType = this.dataExplorerGraphTypes[0].value;
     }
     this.isDataExplorerScatterPlotRegressionLineEnabled = this.componentContent.isDataExplorerScatterPlotRegressionLineEnabled;
-    if (this.componentContent.numDataExplorerYAxis > 1) {
-      this.dataExplorerYAxisLabels = Array(this.componentContent.numDataExplorerYAxis).fill('');
-    }
+    this.dataExplorerYAxisLabels = Array(this.componentContent.numDataExplorerYAxis).fill('');
     this.dataExplorerSeriesParams = this.componentContent.dataExplorerSeriesParams;
   }
 
+  setDataExplorerFixedAxisToColumn(): void {
+    for (
+      let dataExplorerSeriesIndex = 0;
+      dataExplorerSeriesIndex < this.dataExplorerSeries.length;
+      dataExplorerSeriesIndex++
```

Comment:
Nit: I think we should just use i instead of dataExplorerSeriesIndex. It makes the code shorter and easier to read. What do you think?